File: Data Structures & Algorithms/concatenation-of-array/submission-1.js
class Solution {
    getConcatenation(nums) {
        let n=nums.length;
        let ans =new Array(n*2);
       
       for(let i=0;i<n;i++){
        ans[i]=nums[i];
        ans[n+i]=nums[i];
       }
       return ans;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int> mpp;
        int size=nums.size();
        for(int n=0;n<size;n++){
            if(mpp.count(target-nums[n])){
            return {mpp[target-nums[n]],n};
            } 
            mpp[nums[n]]=n;
        }
        
        return {-1,-1};
    } 
};
